etc/parameters/default.json

```diff
@@ -248,51 +248,51 @@
     }
   },
   "current_minimizer_parameters": {
-    "allowed_current": 0.1,
-    "minimum_reached_hysteresis": 0.05,
-    "optimization_speed": 0.0000012,
+    "allowed_current": 0.13,
+    "minimum_reached_hysteresis": 0.05000000074505806,
     "position_difference_threshold": 0.0,
-    "reset_base_offset": 0.0002,
-    "reset_speed": 0.0006,
-    "reset_threshold": 0.0007,
     "optimization_sign": {
       "head": {
-        "yaw": 1,
-        "pitch": 1
+        "pitch": 1.0,
+        "yaw": 1.0
       },
       "left_arm": {
-        "shoulder_pitch": 1,
-        "shoulder_roll": 1,
-        "elbow_yaw": 1,
-        "elbow_roll": 1,
-        "wrist_yaw": 1,
-        "hand": 1
-      },
-      "right_arm": {
-        "shoulder_pitch": 1,
-        "shoulder_roll": -1,
-        "elbow_yaw": 1,
-        "elbow_roll": 1,
-        "wrist_yaw": 1,
-        "hand": 1
+        "elbow_roll": 1.0,
+        "elbow_yaw": 1.0,
+        "hand": 1.0,
+        "shoulder_pitch": 1.0,
+        "shoulder_roll": 1.0,
+        "wrist_yaw": 1.0
       },
       "left_leg": {
-        "hip_yaw_pitch": -1,
-        "hip_roll": -1,
-        "hip_pitch": 1,
-        "knee_pitch": 1,
-        "ankle_pitch": -1,
-        "ankle_roll": -1
+        "ankle_pitch": -1.0,
+        "ankle_roll": -1.0,
+        "hip_pitch": -1.0,
+        "hip_roll": -1.0,
+        "hip_yaw_pitch": -1.0,
+        "knee_pitch": 1.0
+      },
+      "right_arm": {
+        "elbow_roll": 1.0,
+        "elbow_yaw": 1.0,
+        "hand": 1.0,
+        "shoulder_pitch": 1.0,
+        "shoulder_roll": -1.0,
+        "wrist_yaw": 1.0
       },
       "right_leg": {
-        "hip_yaw_pitch": -1,
-        "hip_roll": 1,
-        "hip_pitch": 1,
-        "knee_pitch": 1,
-        "ankle_pitch": -1,
-        "ankle_roll": 1
+        "ankle_pitch": -1.0,
+        "ankle_roll": 1.0,
+        "hip_pitch": -1.0,
+        "hip_roll": 1.0,
+        "hip_yaw_pitch": -1.0,
+        "knee_pitch": 1.0
       }
-    }
+    },
+    "optimization_speed": 1.2000000424450263e-6,
+    "reset_base_offset": 0.00019999999494757503,
+    "reset_speed": 0.0006000000284984708,
+    "reset_threshold": 0.000699999975040555
```

Comment:
please round a few digits

Reply:
I even think, you did not change them intentionally they are in fact the same as in main